dbt/adapters/base/impl.py

```diff
@@ -314,14 +321,10 @@ def behavior(self, flags: List[BehaviorFlag]) -> None:
     def _behavior_flags(self) -> List[BehaviorFlag]:
         """
         This method should be overwritten by adapter maintainers to provide platform-specific flags
+
+        The BaseAdapter should NOT include any global flags here as should be defined in BaseAdapter.GLOBAL_BEHAVIOR_FLAGS
```

Comment:
We should update this based on the chosen approach (module variable or directly in the init.

Reply:
Updated!